deepchecks/nlp/utils/embeddings_calculator.py

```diff
@@ -0,0 +1,85 @@
+# ----------------------------------------------------------------------------
+# Copyright (C) 2021-2022 Deepchecks (https://www.deepchecks.com)
+#
+# This file is part of Deepchecks.
+# Deepchecks is distributed under the terms of the GNU Affero General
+# Public License (version 3 or later).
+# You should have received a copy of the GNU Affero General Public License
+# along with Deepchecks.  If not, see <http://www.gnu.org/licenses/>.
+# ----------------------------------------------------------------------------
+#
+"""Utils module for calculating embeddings for text."""
+
+import pandas as pd
+from tqdm import tqdm
+
+
+def calculate_embeddings_for_text(text: pd.Series, model: str = 'miniLM', file_path: str = 'embeddings.csv') -> pd.DataFrame:
+    """
+    Get default embeddings for the dataset.
+
+    Parameters
+    ----------
+    text : pd.Series
+        The text to get embeddings for.
+    model : str, default 'miniLM'
+        The type of embeddings to return. Can be either 'miniLM' or 'open_ai'.
+        For 'open_ai' option, the model used is 'text-embedding-ada-002' and requires to first set an open ai api key
+        by using the command openai.api_key = YOUR_API_KEY
+    file_path : str, default 'embeddings.csv'
+        If given, the embeddings will be saved to the given file path.
+
+    Returns
+    -------
+        pd.DataFrame
+            The embeddings for the dataset.
+    """
```

Comment:
maybe check if file_path exists, and if so, raise an error?

Reply:
It tries to create the dir if it does not exist

Reply:
I mean that if a user already has that file, don't overwrite it